data-small/mode-share-sites-baseline.csv

```diff
@@ -1,36 +1,38 @@
-site_name,walk_base,cycle_base,drive_base,other_base
-allerton-bywater,23,7,389,120
-ashton-park,301,72,1669,295
-aylesbury,278,53,1949,448
-aylesham,99,15,676,170
-bailrigg,518,145,1513,517
-bath-riverside,708,85,616,405
-bicester,508,219,4276,911
-castlemead,62,15,435,74
-chapelford,99,42,1916,270
-clackers-brook,66,27,462,66
-culm,162,36,1141,219
-dickens-heath,55,17,1416,239
-didcot,278,133,2123,494
-dunton-hills,201,53,2397,1082
-ebbsfleet,337,70,3735,2314
-great-kneighton,340,673,753,402
-halsnead,121,20,957,334
-hampton,488,374,4518,1246
-handforth,137,35,1109,221
-kidbrooke-village,208,148,1184,2365
-lcid,206,10,174,160
-long-marston,149,43,2402,321
-micklefield,11,2,204,55
-newcastle-great-park,284,90,2489,1170
-poundbury,422,57,1204,188
-priors-hall,466,124,3385,656
-taunton-firepool,259,84,397,96
-taunton-garden,561,393,2903,517
-tresham,143,29,1049,163
-trumpington-meadows,112,243,551,187
-tyersal-lane,27,3,129,57
-upton,40,24,1209,186
-wichelstowe,640,165,2955,645
-wixams,250,115,3188,642
-wynyard,111,23,1950,264
+site_name,walk_base,cycle_base,drive_base,other_base
+allerton-bywater,22,6,396,129
+ashton-park,303,72,1680,291
+aylesbury,275,56,1947,448
+aylesham,101,15,678,170
+bailrigg,521,144,1516,513
+bath-riverside,712,87,626,394
+bicester,509,219,4280,910
+castlemead,65,17,440,69
+chapelford,100,47,1929,258
+clackers-brook,69,30,466,61
+culm,166,39,1146,214
+dickens-heath,55,17,1427,242
+didcot,285,137,2130,485
+dunton-hills,203,50,2402,1079
+ebbsfleet,335,69,3716,2316
+great-kneighton,337,669,747,409
+halsnead,122,23,969,328
+hampton,488,371,4514,1249
+handforth,137,35,1109,221
+kidbrooke-village,208,148,1184,2365
+lcid,262,12,181,172
+long-marston,149,43,2402,321
+micklefield,13,5,228,66
+newcastle-great-park,287,93,2509,1162
+poundbury,490,71,1473,219
+priors-hall,463,123,3383,657
+taunton-firepool,261,85,398,95
+taunton-garden,559,392,2899,519
+tresham,145,30,1053,161
+trumpington-meadows,116,246,550,182
+tyersal-lane,43,4,155,90
+upton,40,24,1216,185
+wichelstowe,643,168,2958,643
+wixams,253,120,3194,638
+wynyard,116,23,1967,256
```

Comment:
They are all in alphabetical order until here. Worth ordering by `site_name` before saving out the .csv file?

Reply:
good spot, this has been fixed in the latest commit